.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required # Required for apt-get build-dep
+dist: trusty
+language: cpp
+compiler: gcc
+
+env:
+  - MUMBLE_QT=qt5 MUMBLE_HOST=x86_64-linux-gnu
```

Comment:
Also Qt 5 is available, the problem is that it is a version prior to 5.4, which added _QStandardPaths::AppDataLocation_.